restoration-ecology.csl

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" default-locale="en-GB" demote-non-dropping-particle="sort-only" version="1.0" class="in-text">
+  <info>
+    <title>Restoration Ecology</title>
+    <id>http://www.zotero.org/styles/restoration-ecology.csl</id>
```

Comment:
This should be `<id>http://www.zotero.org/styles/restoration-ecology</id>` (no ".csl" extension)